presto-main/src/main/java/com/facebook/presto/sql/analyzer/FeaturesConfig.java

```diff
@@ -107,7 +107,7 @@
     private double spillMaxUsedSpaceThreshold = 0.9;
     private boolean iterativeOptimizerEnabled = true;
     private boolean enableStatsCalculator = true;
-    private boolean ignoreStatsCalculatorFailures = true;
```

Comment:
My thoughts were basically that this flag must be enabled explicitly in Production, and must be disabled on all possible test environments. As we don't expect it to fail on a regular basis. Otherwise we are risking missing bugs, same as we did with logging plans. Logging plans was under a try-catch, and when at some point it was broken for every query - nobody has noticed.